hw/ip/clkmgr/doc/_index.md

```diff
@@ -77,7 +81,11 @@ This includes but is not limited to:
 * Turning off fabric / gasket clocks, and thus having no way to access the fabric and resume the clock.
 * Turning off memory clocks such that there is no way to execute code that would resume the clocks.
 
+<<<<<<< HEAD
 For this group, there is no reason to allow software control over the clocks, as it could be used to create a system deadlock where after disabling infrastructure clocks there is no way to turn them back on.
+=======
+For this group, there is no reason to allow software control over the clocks, as it could be used to commit suicide.
+>>>>>>> f8b9c9f7 ([clkmgr] clkmgr markdown creation)
```

Comment:
this one I think still has to be fixed.

Reply:
argh... how did i miss that.

thanks for catching.